### DISCUSSION TITLE
Do I need a public IP for just run a polygon node? Not really want to be a validator，just want to sync the blockchain data.

### DISCUSSION THREAD

**Discussion by PenryZen** [maticnetwork/heimdall#1109] [Q&A]
I try to run a node on my local machine, but I get "Error dialing seed" Error when I run the heimdall program, so I wonder why it failed. Is it because the machine which runs the heimdall need a public IP?

**Comment by pratikspatil024** [comment]
Hi @PenryZen - please follow the instructions mentioned [here](https://wiki.polygon.technology/docs/pos/operate/node/full-node-packages/) to run a node using packages. Also, the information about default ports and tags is available [here](https://wiki.polygon.technology/docs/pos/reference/default-ports/).

If the `Error dialing seed` error, still exists, try to delete `addrbook.json` and then add persistent peers and restart.
